tests/unit/basic_crawler/test_basic_crawler.py

```diff
@@ -645,6 +645,17 @@ async def handler(context: BasicCrawlingContext) -> None:
     assert exported_json_str == expected_json_str
 
 
+async def test_crawler_push_data_over_limit() -> None:
+    crawler = BasicCrawler()
+
+    @crawler.router.default_handler
+    async def handler(context: BasicCrawlingContext) -> None:
+        await context.push_data({'hello': 'world' * 3 * 1024 * 1024})
```

Comment:
Why these numbers?
 len(('world' * 3 * 1024 * 1024).encode('utf-8')) is 15728640

Shouldn't this be somehow linked to the 9 mb defined in https://github.com/apify/crawlee-python/blob/e26a32383244d3972d3bd7621e7ae7aabf9a39c9/src/crawlee/storages/_dataset.py#L185 ?

Reply:
The important thing is that it's longer than the limit. Relying on constants imported from the code that's being tested is not great. We may want to explain this in a comment.

Reply:
I would expect both the test and code to import this currently unnamed "9" constant from somewhere. Test would then use something that is "imported_9+1". Such test would be more self explanatory and more resistant to change.

Reply:
> We may want to explain this in a comment.

Yes please